### DISCUSSION TITLE
FYI: gprbuild as a linter

### DISCUSSION THREAD

**Discussion by TamaMcGlinn** [krischik/vim-ada#41] [Ideas]
Not an actual issue, but this may interest users of the plugin. It's possible to configure gprbuild to be a linter in NeoVim so that you get immediate feedback when your syntax is incorrect:

![Screenshot from 2022-11-17 12-53-27](https://user-images.githubusercontent.com/5008897/202439617-185801cc-f16f-4df5-b49b-edee16d0ec5f.png)

configuration instructions are described [here](https://github.com/mattn/efm-langserver/issues/236); I think it's out of scope to include in vim-ada, but it might become a plugin of its own.

**Comment by krischik** [comment]
I use GVim because I like to have graphical user interface. I have to check of integrating of language server protocol is feasible. But that might take some time. I move this to discussions for later consideration.
